### DISCUSSION TITLE
win10下最新版无法打开设置界面。win7下也是如此，不过win7用不了ocr功能，win10可以用。

### DISCUSSION THREAD

**Discussion by a137933** [shuax/MouseInc.Settings#340] [Q&A]
![image](https://user-images.githubusercontent.com/34773099/187197537-c9e5cccb-2413-4396-9d85-8acee51ac1d6.png)

重新下载过好几次，没用

**Comment by chenpiqqqq** [comment]
我也是这个问题，才到这里看看怎么情况

**Comment by Eunknight** [comment]
https://github.com/shuax/MouseInc.Settings/discussions/335#discussioncomment-3523123
